core/src/com/unciv/logic/civilization/managers/TechManager.kt

```diff
@@ -119,7 +120,10 @@ class TechManager : IsPartOfGameInfoSerialization {
         techCost /= getScienceModifier(techName)
         val mapSizePredef = civInfo.gameInfo.tileMap.mapParameters.mapSize.getPredefinedOrNextSmaller()
         techCost *= mapSizePredef.techCostMultiplier
-        techCost *= 1 + (civInfo.cities.size - 1) * mapSizePredef.techCostPerCityModifier
+        var cityModifier = (civInfo.cities.size - 1) * mapSizePredef.techCostPerCityModifier
+        for (unique in civInfo.getMatchingUniques(UniqueType.LessTechCostFromCities)) cityModifier *= 1 - unique.params[0].toFloat() / 100
```

Comment:
This should definitely be adjacent to the multiplication, the second unique interferes in the middle for no reason
Not a blocker but yes annoying to look at